homeassistant/components/group/registry.py

```diff
@@ -39,32 +39,38 @@ def _process_group_platform(
     hass: HomeAssistant, domain: str, platform: GroupProtocol
 ) -> None:
     """Process a group platform."""
-    current_domain.set(domain)
     registry: GroupIntegrationRegistry = hass.data[REG_KEY]
     platform.async_describe_on_off_states(hass, registry)
 
 
 class GroupIntegrationRegistry:
     """Class to hold a registry of integrations."""
 
-    def __init__(self) -> None:
+    def __init__(self, hass: HomeAssistant) -> None:
         """Imitialize registry."""
+        self.hass = hass
         self.on_off_mapping: dict[str, str] = {STATE_ON: STATE_OFF}
         self.off_on_mapping: dict[str, str] = {STATE_OFF: STATE_ON}
         self.on_states_by_domain: dict[str, set[str]] = {}
         self.exclude_domains: set[str] = set()
+        self.state_group_mapping: dict[str, tuple[str, str]] = {}
+        self.group_entities: set[Group] = set()
```

Comment:
I missed the shared container for entities when doing the review. Its not actually used in this PR